### DISCUSSION TITLE
there is no depotdownloader.dll

### DISCUSSION THREAD

**Discussion by aguest12** [SteamRE/DepotDownloader#413] [Q&A]
![Screenshot_66](https://user-images.githubusercontent.com/110438477/182277369-80f9b1fa-f427-45af-9970-5d328a33e1b5.png)
tried to use it but it didnt see any depotloader.dll

**Comment by yaakov-h** [comment]
It's definitely there in the ZIP. Are you running from a directory containing the extracted contents, or from some other working directory?

**Comment by aguest12** [comment]
i just realized i was downloading the code not the zip

**Comment by aguest12** [comment]
it wont go past %8.74 bad gateway

**Comment by Beeque** [comment]
no dll in zip file either?
